### PR TITLE
This is  404 not found page

### PR DESCRIPTION
**Pull request by thanhsondev**
This is  404 not found page. Websites often use the HTTP error code 404 to tell users that the page they tried to access could not be found on the server. This could be for a variety of reasons, including the URL being wrong, the page being deleted or moved, or there being a technical error on the site.

### CODE REVIEW

**Comment by luanvo1910** [comment]
chụp thêm hình ảnh page cho mn coi với

File: src/pages/404notfound/PageNotFound.jsx
Comment (luanvo1910) [inline src/pages/404notfound/PageNotFound.jsx:1]: sao ko lấy thẳng url của ảnh mà fetch chi rườm rà z :v

File: .idea/.gitignore
Comment (luanvo1910) [inline .idea/.gitignore:1]: mấy file trong .idea là gì v, sao tạo thêm 1 file .gitignore nữa

**Review by luanvo1910 — CHANGES_REQUESTED** [review]

**Comment by thanhsondev** [comment]
\
Screenshot 2023-12-12 090810

**Pull request closed without merging**